cmd/print.go

```diff
@@ -245,7 +245,7 @@ func PrintProgressBarsOnEvents(w io.Writer, bus event.Bus) {
 	progress := map[string]*mpb.Bar{}
 
 	if bus == nil {
-		log.Errorf("event bus is nil")
+		log.Debugf("event bus is nil")
```

Comment:
this was logging everytime we ran an rpc call... moved it to debug for better user experience

Reply:
yep. 99% of logging should be `Debugf`